data-pipeline/src/pipeline/database.py

```diff
@@ -71,69 +209,7 @@ def upsert(
     update_stmt = f'MERGE {table_name} as dest USING {temp_table} as src ON {" AND ".join(match_keys)}  WHEN MATCHED THEN UPDATE SET {", ".join(update_cols)} WHEN NOT MATCHED BY TARGET THEN INSERT ({", ".join(insert_cols)}) VALUES ({", ".join(insert_vals)});'
     with engine.begin() as cnx:
         cnx.execute(sqlalchemy.text(update_stmt))
-        cnx.execute(sqlalchemy.text(f"DROP TABLE IF EXISTS {temp_table}"))
-
-
-def insert_comparator_set(
```

Comment:
I've just shunted these two functions to the bottom of this module; the function definitions then roughly follow the order in which they're called.